From ca0ac8836c89d2ac59fe84034ca9db98cfeb163a Mon Sep 17 00:00:00 2001
From: Dennis Weissmann <github@dennisweissmann.me>
Date: Sun, 17 Sep 2017 16:35:41 +0200
Subject: [PATCH] include iPhone 8(+) and X in the generated Swift file

---
 DeviceKit.podspec | 4 ++--
 Source/Info.plist | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/DeviceKit.podspec b/DeviceKit.podspec
index a50ea27..830450f 100644
--- a/DeviceKit.podspec
+++ b/DeviceKit.podspec
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'DeviceKit'
-  s.version      = '1.3.0'
+  s.version      = '1.3.1'
   s.summary      = 'DeviceKit is a µ-framework that provides a value-type replacement of UIDevice.'
 
   s.description             = <<-DESC
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target  = '9.0'
 
   s.source                  = { :git => 'https://github.com/dennisweissmann/DeviceKit.git', :tag => s.version }
-  s.source_files              = 'Source/Device.generated.swift'
+  s.source_files            = 'Source/Device.generated.swift'
 
   s.requires_arc = true
 end
diff --git a/Source/Info.plist b/Source/Info.plist
index 9f4a4aa..aa82cca 100644
--- a/Source/Info.plist
+++ b/Source/Info.plist
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.3.0</string>
+	<string>1.3.1</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
--
Working Copy 3.4.3

